terraform/modules/vpc/nat.tf

```diff
@@ -5,10 +5,10 @@ resource "aws_eip" "nat" {
 
```

Comment:
@Nouman72884 I think it better if you create route table and its association in single file, and add routes in multiple files where needed.